common-util/Details/ServiceDetailsHelper.jsx

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable react/prop-types */
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  Button, Typography, Input, notification, Form,
+} from 'antd/lib';
+import { DynamicFieldsForm } from 'common-util/DynamicFieldsForm';
+import {
+  checkIfServiceIsWhitelisted,
+  setOperatorsStatusesRequest,
+} from './ServiceState/utils';
+
+const { Text } = Typography;
+
+export const OperatorWhitelist = ({ isWhiteListed, setOpWhitelist, id }) => {
+  const account = useSelector((state) => state?.setup?.account);
+  const [form] = Form.useForm();
+
+  const [isCheckLoading, setIsCheckLoading] = useState(false);
+
+  useEffect(() => {
+    if (id) {
+      setOpWhitelist();
+    }
+  }, [id]);
+
+  const onCheck = async (values) => {
+    try {
+      setIsCheckLoading(true);
+      const isValid = await checkIfServiceIsWhitelisted(
+        id,
+        values.operatorAddress,
+      );
+
+      const message = `Operator ${values.operatorAddress} is ${
+        isValid ? '' : 'not'
+      } whitelisted`;
+      notification.success({ message });
+    } catch (error) {
+      console.error(error);
+    } finally {
+      setIsCheckLoading(false);
+    }
+  };
+
+  return (
+    <>
+      {isWhiteListed && (
+        <>
+          <Text>Check if Operator Address is whitelisted?</Text>
```

Comment:
This text is redundant, delete

Reply:
Yes that's fair. In that case let's change the button to "Check whitelist status" @mohandast52

Reply:
(and delete this line)